ios/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+ruby '~> 2.7.0'
+gem "fastlane"
+gem 'cocoapods', '1.14.2'
```

Comment:
Awesome! Just ran into this, where my installed cocoapods version was different.